Engine/DataFeeds/Subscription.cs

```diff
@@ -180,7 +180,7 @@ public bool RemoveSubscriptionRequest(Universe universe = null)
                 // if the security is no longer a member of the universe, then mark the subscription properly
                 // universe may be null for internal currency conversion feeds
                 // TODO : Put currency feeds in their own internal universe
-                if (!removedUniverses.Any(x => x.Members.ContainsKey(Configuration.Symbol)))
+                if (!removedUniverses.Any(x => x.Securities.ContainsKey(Configuration.Symbol)))
```

Comment:
Why is this?

Reply:
`Members` is created calling `ToDictionary` on `Securities`, objective being skip the `ToDictionary` call completly 😃

Reply:
Another nice change!

Here's what `Members` looks like:

``` csharp
public Dictionary<Symbol, Security> Members
{
    get { return Securities.Select(x => x.Value.Security).ToDictionary(x => x.Symbol); }
}
```

whereas `Securities`:

``` csharp
internal virtual ConcurrentDictionary<Symbol, Member> Securities
{
    get;
    private set;
}
```